<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Macro_Editor</class>
 <widget class="QDialog" name="Macro_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macro Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>#1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_1_name"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="macro_1_content"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>#2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_2_name"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="macro_2_content"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>#3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_3_name"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="macro_3_content"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>#4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_4_name"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="macro_4_content"/>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>#5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_5_name"/>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="macro_5_content"/>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>#6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_6_name"/>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="macro_6_content"/>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>#7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_7_name"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="macro_7_content"/>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>#8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_8_name"/>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="macro_8_content"/>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>#9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_9_name"/>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="macro_9_content"/>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>#10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_10_name"/>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="macro_10_content"/>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>#11</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_11_name"/>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="macro_11_content"/>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>#12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="macro_12_name"/>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="macro_12_content"/>
   </item>
   <item row="12" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
